primitives/src/conststr.rs

```diff
@@ -469,139 +470,6 @@ impl TryFrom<String> for InlinedString {
     }
 }
 
-mod ffi {
```

Comment:
I have plans to restore this. I removed it because `Account::roles` returns `Vec<&RoleId`> which is translated into `Vec<&ConstString>`. This type cannot be returned from an FFI function at the moment because it would pass on ownership. In the #2978 I am introducing limited ownership transfer because it was too difficult (in some situations even impossible) to generate a wrapper API if there is no ownership transfer.

I maintain that this is not a particularly important piece of code and can be restored